spec/requests/search_controller_spec.rb

```diff
@@ -231,6 +231,11 @@
       expect(SearchLog.where(term: 'wookie')).to be_blank
     end
 
+    it "does not raise 500" do
```

Comment:
I think this should be more descriptive of what params should not result in 500 being returned.

Reply:
If the search term includes only advanced filters, after all of these will be processed, the term becomes empty.